src/lib/Cashlink.ts

```diff
@@ -110,12 +123,14 @@ class Cashlink {
     private _eventListeners: {[type: string]: Array<(data: any) => void>} = {};
     private _messageBytes: Uint8Array = new Uint8Array(0);
     private _value: number | null = null;
+    private _fee: number | null = null;
     private _knownTransactions: DetailedPlainTransaction[] = [];
 
     constructor(
         public keyPair: Nimiq.KeyPair,
         public address: Nimiq.Address,
         value?: number,
+        fee?: number,
```

Comment:
Optional parameters usually come at the end of the argument list, to not have to write `undefined` in the middle of the function call / object creation...

Reply:
All following parameters are optional.